app/pb_kits/playbook/pb_title/title.rb

```diff
@@ -2,77 +2,30 @@
 
 module Playbook
   module PbTitle
-    class Title < Playbook::PbKit::Base
-      PROPS = %i[configured_aria
-                 configured_classname
-                 configured_dark
-                 configured_data
-                 configured_id
-                 configured_size
-                 configured_tag
-                 configured_text].freeze
+    class Title
+      include Playbook::Props
 
-      def initialize(aria: default_configuration,
-                     classname: default_configuration,
-                     dark: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     size: default_configuration,
-                     tag: default_configuration,
-                     text: default_configuration)
-        self.configured_aria = aria
-        self.configured_classname = classname
-        self.configured_dark = dark
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_size = size
-        self.configured_tag = tag
-        self.configured_text = text
-      end
-
-      def tag
-        tag_options = %w[h1 h2 h3 h4 h5 h6 p div span]
-        one_of_value(configured_tag, tag_options, "h3")
-      end
-
-      def text
-        default_value(configured_text, "Title text")
-      end
-
-      def size
-        size_options = [1, 2, 3, 4]
-        one_of_value(configured_size, size_options, 3)
-      end
+      partial "pb_title/title"
 
-      def size_class
-        adjusted_value(size, size, nil)
-      end
-
-      def dark_class
-        true_value(configured_dark, "dark", nil)
-      end
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :size, type: Playbook::Props::Enum,
+                  values: %w[1 2 3 4],
```

Comment:
We definitely want to get this up in a local server before merging. `size` looks to accept integers 1-4, but we are declaring that it be strings "1"-"4". I'm wondering if the examples in the docs blow up...

Reply:
Just remove `%w` here and separate the values with `,` and it should be fine.

Reply:
Resolved. The fix required changing several .slim files and a helper method use of pb_title's size value from a string to an integer.